Interaction region for <video> play button is not round

hortont424 opened the pull request:
#### fc190ad09c4664375ffba90c722a6049969b60c0
<pre>
Interaction region for &lt;video&gt; play button is not round
<a href="https://bugs.webkit.org/show_bug.cgi?id=242831">https://bugs.webkit.org/show_bug.cgi?id=242831</a>

Reviewed by Dean Jackson and Devin Rousso.

* LayoutTests/interaction-region/paused-video-regions-expected.txt: Added.
* LayoutTests/interaction-region/paused-video-regions.html: Added.
Add a test that dumps the interaction region for the play button.
Previously, it would not have had a non-zero radius.

* Source/WebCore/Modules/modern-media-controls/controls/button.css:
(button.center,):
(button.center &gt; .background-tint,): Deleted.
Apply the circular border radius to the &lt;button&gt; as well as its children,
in order to make its interaction region correct.

This has no impact on actual functionality, since the video itself and the button
can both be clicked to start playback.

Canonical link: <a href="https://commits.webkit.org/252594@main">https://commits.webkit.org/252594@main</a>
</pre>

dcrousso commented on Source/WebCore/Modules/modern-media-controls/controls/button.css:
```diff
@@ -106,6 +106,8 @@ button.small-center {
 }
 
+button.center,
+button.small-center,
```
NIT: we can simplify this a bit :)
```
button:is(.center, .small-center),
button:is(.center, .small-center) > .background-tint,
button:is(.center, .small-center) > .background-tint > div {
```

hortont424 committed:
Interaction region for <video> play button is not round
https://bugs.webkit.org/show_bug.cgi?id=242831

Reviewed by Dean Jackson and Devin Rousso.

* LayoutTests/interaction-region/paused-video-regions-expected.txt: Added.
* LayoutTests/interaction-region/paused-video-regions.html: Added.
Add a test that dumps the interaction region for the play button.
Previously, it would not have had a non-zero radius.

* Source/WebCore/Modules/modern-media-controls/controls/button.css:
(button.center,):
(button.center > .background-tint,): Deleted.
Apply the circular border radius to the <button> as well as its children,
in order to make its interaction region correct.

This has no impact on actual functionality, since the video itself and the button
can both be clicked to start playback.

Canonical link: https://commits.webkit.org/252594@main
